geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -325,13 +325,15 @@ def includeme(config: pyramid.config.Configurator):
     get_user_from_request = create_get_user_from_request(settings)
     config.add_request_method(get_user_from_request, name="user", property=True)
     config.add_request_method(get_user_from_request, name="get_user")
-    # Be able for an organisation to override the method to use alternate roles name for
-    # the standard roles 'anonymous', 'registred' and 'intranet'.
+    # Be able for an organisation to override the method to use alternate:
```

Comment:
Missplaced `:` ? it looks weird here
```suggestion
    # Be able for an organisation to override the method to use alternate
```

Reply:
My idea is to introduce the 3 methods as a list then I add a dash to introduce the elements...

Reply:
It's hard to understand like that. At least for me